<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget_Terminology_mangae</class>
 <widget class="QWidget" name="Widget_Terminology_mangae">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>493</width>
    <height>242</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListView" name="listView_Terminologylist"/>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_Terminologydescripte">
       <property name="title">
        <string>术语库预览</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_Terminologyname">
          <property name="text">
           <string>术语库名</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="lineEdit_Terminologyname"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_Terminologynumber">
          <property name="text">
           <string>条目数量</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="lineEdit_Terminologynumber"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_Terminologycreatetime">
          <property name="text">
           <string>创建时间</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lineEdit_Terminologycreatetime"/>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_owner">
          <property name="text">
           <string>创建者</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_owner"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_Terminologydescription">
          <property name="text">
           <string>描述</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QTextEdit" name="textEdit_description"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_createTerminology">
       <property name="text">
        <string>新建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_edit">
       <property name="text">
        <string>编辑</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_deleteTerminology">
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
